packages/core/src/new-api/internal/deployer.ts

```diff
@@ -30,6 +30,8 @@ import { assertIgnitionInvariant } from "./utils/assertions";
 import { getFuturesFromModule } from "./utils/get-futures-from-module";
 import { validate } from "./validation/validate";
 
+const BLOCK_POLLING_INTERVAL = 200;
```

Comment:
Should be received by `deploy`, which means it'd be configurable by the plugin.

Reply:
I have recorded a todo for this. It has been partially done in the timeouts PR: https://github.com/NomicFoundation/ignition/pull/306

But I have added an overall todo to check that config is being passed through the entire stack and comes from the plugin: https://github.com/NomicFoundation/ignition/issues/312